configs/server/forseti_conf_server.yaml.in

```diff
@@ -281,6 +281,10 @@ scanner:
 ##############################################################################
 
 notifier:
+    api_quota:
+        securitycenter:
+            max_calls: 14
```

Comment:
just double-check, this is higher than what is defined above which is 1.1

Reply:
I took 1000 qpm for the write API and divided by 60 which got me 16 and change.  I then tweaked it until I stopped getting 429's.

I can lower it to 1 query per second.  This will give us 60 qpm which is safely below the 1000 qpm quota.

Reply:
IIRC, we started using this API during alpha, when the supported quota was very low.  So, what you have should be fine.